arcgis-ios-sdk-samples/Route & Directions/Find closest facility to multiple incidents (service)/FindClosestFacilityMultipleIncidentsServiceViewController.swift

```diff
@@ -0,0 +1,226 @@
+//
+// Copyright © 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+/// A view controller that manages the interface of the Find Closest Facility to
+/// Multiple Incidents (Service) sample.
+class FindClosestFacilityMultipleIncidentsServiceViewController: UIViewController {
+    /// The map view managed by the view controller.
+    @IBOutlet weak var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+            mapView.graphicsOverlays.add(routesOverlay)
+        }
+    }
+    /// The bar button item that initiates the solve route operation.
+    @IBOutlet weak var solveRoutesButtonItem: UIBarButtonItem!
+    /// The bar button item that removes the solved routes.
+    @IBOutlet weak var resetButtonItem: UIBarButtonItem!
+    
+    /// The graphics overlay for the routes.
+    let routesOverlay = AGSGraphicsOverlay()
+    
+    /// The facility features.
+    var facilityFeatures = [AGSFeature]()
+    /// The incident features.
+    var incidentFeatures = [AGSFeature]()
+    /// The task used to find the closest facilities.
+    let closestFacilityTask: AGSClosestFacilityTask = {
+        let url = URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ClosestFacility")!
+       return AGSClosestFacilityTask(url: url)
```

Comment:
fix indentation

Reply:
Thanks for noticing that! I'll take care of it.